cost/azure/csp/cheaper_regions/azure_cheaper_regions.pt

```diff
@@ -0,0 +1,241 @@
+      billing_center_names = _.map(param_billing_centers, function(name){ return name.toLowerCase(); });
+      billing_center_ids = _.compact(_.map(top_billing_centers, function(value){ if(_.contains(billing_center_names, value.name.toLowerCase())){return value.id} }));
+    }
+    console.log('billing_centers:'+JSON.stringify(billing_centers))
+    console.log('billing_centers_ids:'+ billing_center_ids)
+        console.log('top_billing_centers:'+ JSON.stringify(top_billing_centers))
+
+    var request = {
+      auth: "auth_rs",
+      verb: "POST",
+      host: "optima.rightscale.com",
+      path: "/bill-analysis/orgs/" + org_id + "/costs/aggregated",
+      body_fields: {
+        "dimensions": ["region","billing_center_id","service"],
+        "granularity": "month",
+        "metrics": ["cost_amortized_unblended_adj"],
+        "billing_center_ids": billing_center_ids,
+        "start_at": start_at,
+        "end_at": end_at
+      },
+      headers: {
+        "Api-Version": "1.0",
+        "User-Agent": "RS Policies",
+      }
+    }
+  EOS
+end
+
+script "js_merge_bc_name", type: "javascript" do
+  parameters "new_bc_costs", "billing_centers"
+  result "results"
+  code <<-EOS
+    var results = []
+    var ds_cheaper_regions = {
+      "regions": {
+        "US West":    "US West 2",
+        "US East":    "US East 2",
+        "KR Central": "KR South",
+        "US Central": "US West 2",
+        "CA Central": "CA East",
+        "EU West":    "UK South",
+        "EU North":   "UK West",
+        "JA East":    "KR South",
+        "JA West":    "KR South"
+        }
+    }
+    for (var i = 0; i < new_bc_costs.length; i++) {
+      console.log(new_bc_costs[i])
+      console.log(new_bc_costs[i].billing_center_id)
+      var arr_bc_name = _.reject(billing_centers, function(bc){ return bc.id != new_bc_costs[i].billing_center_id });
+      var bc_name = arr_bc_name[0].name
+      var new_region = ds_cheaper_regions["regions"][new_bc_costs[i].region]
+      console.log(new_region)
+      console.log(bc_name)
+      if (new_region != null && new_region !== undefined ) {
+        results.push({
+          billing_center_name: bc_name,
+          billing_center_id: new_bc_costs[i].billing_center_id,
+          region: new_bc_costs[i].region,
+          new_region: new_region,
+          service: new_bc_costs[i].service
+        })
+      }else{
+        continue
+      }
+    }
+EOS
+end
+
+###############################################################################
+# Policy
+###############################################################################
+
+policy "azure_cheaper_regions" do
+  validate_each $ds_new_bc_cost_name do
+    summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} rows with cheaper data"
+    detail_template <<-EOS
+# Cheaper Regions
+| Billing Center Name | Billing Center ID | Region | New Region | Service |
```

Comment:
I suggest changing Region to  `Region in use` and New Region to `Suggested Region` or something like that.